x-pack/plugins/reporting/server/export_types/csv_searchsource/execute_job.ts

```diff
@@ -19,7 +18,7 @@ export const runTaskFnFactory: RunTaskFnFactory<RunTaskFn<TaskPayloadCSV>> = (
   const config = reporting.getConfig();
 
   return async function runTask(jobId, job, cancellationToken, stream) {
-    const logger = parentLogger.clone([CSV_JOB_TYPE, 'execute-job', jobId]);
```

Comment:
The extra `*_JOB_TYPE` tag was redundant: the job type was already being added in the parentLogger that is passed in.